packages/core/package.json

```diff
@@ -24,12 +24,13 @@
     "typescript-standard": "0.3.36"
   },
   "dependencies": {
-    "wot-typescript-definitions": "0.7.3",
     "@node-wot/td-tools": "0.7.2",
-    "uuid": "3.4.0",
+    "@types/uritemplate": "0.3.4",
     "rxjs": "5.5.11",
     "uritemplate": "0.3.4",
-    "@types/uritemplate": "0.3.4"
+    "uuid": "3.4.0",
+    "vm2": "^3.9.2",
+    "wot-typescript-definitions": "0.7.3"
```

Comment:
May I ask you to introduce back the original order since most of the entries are just re-shuffled and not *new* expect "vm2": "^3.9.2"

The reason I am asking for it is that if we merge THIS work into https://github.com/eclipse/thingweb.node-wot/tree/v0.8.x we will get merge problems since this will be detected as a CHANGE.

Reply:
yes sure I'll try to put everything back in order. It was npm that changed the order.

Reply:
It should be fine now 👍